tests/v2/validation/rbac/rbac_additional_test.go

```diff
@@ -60,116 +52,131 @@ func (rb *RBACAdditionalTestSuite) SetupSuite() {
+	require.NoError(rb.T(), errUserRole)
+	standardUserClient, err = standardUserClient.ReLogin()
 	require.NoError(rb.T(), err)
+	return standardUserClient, createProjectAsAdmin
+}
 
-	errUserRole2 := users.AddProjectMember(rb.additionalUserClient, rb.standardUserCOProject, addNewUserAsPO, roleProjectOwner, nil)
-	require.NoError(rb.T(), errUserRole2)
+func (rb *RBACAdditionalTestSuite) TestProjectOwnerCanAddProjectOwner() {
+	subSession := rb.session.NewSession()
+	defer subSession.Cleanup()
 
-	addNewUserAsPOClient, err = addNewUserAsPOClient.ReLogin()
+	standardUserClient, adminProject := rb.testSetupUserAndProject()
+	rb.T().Logf("Creating an additional standard user")
+	additionalUser, additionalUserClient, err := setupUser(rb.client, standardUser)
 	require.NoError(rb.T(), err)
 
-	userGetProject, err := projects.GetProjectList(addNewUserAsPOClient, rb.cluster.ID)
+	log.Info("Validating if Project Owner can add another Project Owner")
+	errUserRole := users.AddProjectMember(standardUserClient, adminProject, additionalUser, roleProjectOwner, nil)
+	require.NoError(rb.T(), errUserRole)
+	additionalUserClient, err = additionalUserClient.ReLogin()
+	require.NoError(rb.T(), err)
+
+	userGetProject, err := projects.GetProjectList(additionalUserClient, rb.cluster.ID)
 	require.NoError(rb.T(), err)
 	assert.Equal(rb.T(), 1, len(userGetProject.Data))
-	assert.Equal(rb.T(), rb.standardUserCOProject.Name, userGetProject.Data[0].Name)
+	assert.Equal(rb.T(), adminProject.Name, userGetProject.Data[0].Name)
 
-	errRemoveMember := users.RemoveProjectMember(rb.standardUserClient, addNewUserAsPO)
+	errRemoveMember := users.RemoveProjectMember(standardUserClient, additionalUser)
 	require.NoError(rb.T(), errRemoveMember)
 
-	userProjectEmptyAfterRemoval, err := projects.GetProjectList(addNewUserAsPOClient, rb.cluster.ID)
+	userProjectEmptyAfterRemoval, err := projects.GetProjectList(additionalUserClient, rb.cluster.ID)
 	require.NoError(rb.T(), err)
 	assert.Equal(rb.T(), 0, len(userProjectEmptyAfterRemoval.Data))
-	users.RemoveProjectMember(rb.additionalUserClient, rb.additionalUser)
 }
 
-func (rb *RBACAdditionalTestSuite) ValidateCannotAddMPMsAsProjectOwner() {
-	createProjectAsCO, err := createProject(rb.client, rb.cluster.ID)
+func (rb *RBACAdditionalTestSuite) TestUserManageProjectMemberCannotAddProjectOwner() {
+	subSession := rb.session.NewSession()
+	defer subSession.Cleanup()
+
+	standardUserClient, adminProject := rb.testSetupUserAndProject()
+	rb.T().Logf("Creating an additional standard user")
+	additionalUser, additionalUserClient, err := setupUser(rb.client, standardUser)
 	require.NoError(rb.T(), err)
-	rb.standardUserCOProject = createProjectAsCO
 
-	log.Info("Validating if Manage Project Member cannot add Project Owner")
-	errUserRole := users.AddProjectMember(rb.standardUserClient, rb.standardUserCOProject, rb.additionalUser, roleCustomManageProjectMember, nil)
+	log.Info("Validating Manage Project Member cannot add Project Owner")
```

Comment:
why use log.info here instead of logrus / testing.T?

Reply:
not sure if its a standard to be followed - using logrus when using testing.T but I can update it because it makes sense to use logrus as its an already provided extension by testing.T instead of using a new extension.

Reply:
oh :) but, do we have rules around when to use testing.T().log vs logrus? I haven't seen them both in the same test before, so I guess i'd have to see the jenkins run to know the difference.